user_interaction/static/themes/dark-theme.css

```diff
@@ -40,13 +40,34 @@
     --greenify-dropdown-hover-text-color: inherit;
     --greenify-dropdown-active-bg-color: var(--greenify-bg-color);
     --greenify-dropdown-active-text-color: var(--greenify-text-color);
+
+    /*Calendar day color*/
+    --fc-neutral-bg-color: green;
 }
 
 /* Lighten link colors */
 a {
     color: #55a7ff;
 }
+a:hover {
+    color: white;
+}
+/* Text color of tabs*/
+.nav-tabs .nav-link.active {
+    color: white;
+}
 
-.alert-info {
-    /* background-color: #243f44; */
+.text-muted {
+    color: whitesmoke !important;
+}
+
+.btn-outline-success.disabled {
+    color: white;
+    background-color: red;
+    font-size: 0px;
+}
+.btn-outline-success.disabled::after {
+    visibility: visible;
+    content: "Registration closed";
```

Comment:
This kind of logic should not be in the CSS. Instead, modify the relevant template in order to change the button text (or even the entire button itself)

Reply:
Then i will delete this